ios/FluentUI.Demo/FluentUI.Demo/Demos/DrawerDemoController.swift

```diff
@@ -143,6 +144,10 @@ class DrawerDemoController: DemoController {
         presentDrawer(sourceView: sender, presentationDirection: .up, contentView: containerForActionViews(), resizingBehavior: .dismissOrExpand)
     }
 
+    @objc private func showBottomDrawerWithExpandStateAndPassthroughBackgroundButtonTapped(sender: UIButton) {
```

Comment:
In landscape mode it is also hard to interact with the underlying view controller. There is just a minimal gap that allows the interaction.

One potential solution I can think of for this scenario is to make the 4 buttons a grid (2x2) instead of a stack of 4 buttons.

Reply:
Fixed!